lib/utils/execution-state-manager.js

```diff
@@ -165,6 +177,17 @@ class ExecutionStateManager {
     this._browserToModuleMap.set(browserId, browserModuleList);
   }
 
+  /**
+   * Pushes the module detail into the moduleRunDetails array
+   *
+   * @param {*} moduleName
+   * @param {*} totalAssertionNum
+   * @param {*} moduleTotalDuration
+   */
+  addModuleDoneDetailToModuleRunDetails(moduleName, totalAssertionNum, moduleTotalDuration) {
```

Comment:
as suggested above, could be renamed & simplified to `addMetadataToModuleMetadata(metadata){`